src/main/java/com/google/cloud/tools/app/impl/cloudsdk/CloudSdkAppEngineStandardStaging.java

```diff
@@ -68,16 +68,16 @@ public void stageStandard(StageStandardConfiguration config) throws AppEngineExc
 
     Path dockerfile =
         config.getDockerfile() == null ? null : config.getDockerfile().toPath();
-    Path dockerfileDestination = config.getStagingDirectory().toPath();
+    Path dockerfileDestination = config.getSourceDirectory().toPath();
```

Comment:
hah, I thought you were talking about the confusion between sourceDir and stagingDir....